dashboard/tests/test_usage_stats_head.py

```diff
@@ -0,0 +1,23 @@
+import sys
+import ray
+import requests
+import pytest
+from ray._raylet import GcsClient
+
+
+@pytest.mark.asyncio
```

Comment:
You can add the test to the existing `test_usage_stats.py`. Check `test_usage_stats_enabled_endpoint`

Reply:
Got it! Let me move the test there